### DISCUSSION TITLE
Trigger Words are Being Unintentionally Abused

### DISCUSSION THREAD

**Discussion by shaneholloman** [civitai/civitai#233] [Ideas]
I believe moving forward when a model contributor at Civitai site, we should be enforcing or at least strongly suggesting that trigger words are unique and only one of them should be allowed and baked into for that particular model, be it a Checkpoint, TI or LoRA

Some contributors a spraying trigger words into there models that interfere with and muddle prompt results

Example this model (if it's yours, thank you for your contribution)
https://civitai.com/models/7563/alice-nikke

ALICE \(NIKKE\)
LATEX BODYSUIT
PINK BODYSUIT
HEADPHONES
TWINTAILS
:D
ANIMAL EAR HEADPHONES
SHINY CLOTHES
SNIPER RIFLE
RIFLE

All of these are going to interfere with other models and trigger them. suggest the trigger is 
`alice-nikke` and thats it done, otherwise we are going to have an unruly mess on our hands

**Comment by WASasquatch** [comment]
You do know some collision is intentional, right, to fine-tune areas of a model? Take "Tinashe" for example. She's in 1.5, but not enough, so little pointy and skinny. I was able to fine-tune a TI of her, using collision, for far better results, in half the training time (then someone not even in the model)
